tensorboard/program_test.py

```diff
@@ -62,6 +62,18 @@ def testMakeServerBlankHost(self):
         self._StubApplication(),
         self.make_flags(host='', port=0, path_prefix=''))
     self.assertStartsWith(server.get_url(), 'http://')
+    
+  def testPathPrefixSlash(self):
+    #Test that checks the path prefix ends with one trailing slash
+    server = program.WerkzeugServer(
+        self._StubApplication(),
+        self.make_flags(host='', port=0, path_prefix='/test'))
+    self.assertEndsWith(server.get_url(), 'http://127.0.0.1:0/test/')
```

Comment:
This should just be `self.assertEndsWith(server.get_url(), '/test/')` - that way you avoid trying to match on the rest of the URL which isn't going to be `127.0.0.1:0`.

Same thing below.

Reply:
Ohk Done!